apps/frontend/package.json

```diff
@@ -0,0 +1,68 @@
+{
+	"dependencies": {
+		"@angular/animations": "^15.2.0",
+		"@angular/cdk": "^15.2.0",
+		"@angular/common": "^15.2.0",
+		"@angular/compiler": "^15.2.0",
+		"@angular/core": "^15.2.0",
+		"@angular/forms": "^15.2.0",
+		"@angular/material": "^15.2.0",
+		"@angular/platform-browser": "^15.2.0",
+		"@angular/platform-browser-dynamic": "^15.2.0",
+		"@angular/router": "^15.2.0",
+		"@ngx-translate/core": "^14.0.0",
+		"rxjs": "~7.8.0",
+		"tslib": "^2.3.0",
+		"zone.js": "~0.12.0"
+	},
+	"devDependencies": {
+		"@angular-devkit/build-angular": "^15.2.0",
+		"@angular-eslint/eslint-plugin": "^15.2.1",
+		"@angular-eslint/eslint-plugin-template": "^15.2.1",
+		"@angular-eslint/template-parser": "^15.2.1",
+		"@angular/cli": "~15.2.0",
+		"@angular/compiler-cli": "^15.2.0",
+		"@bartholomej/ngx-translate-extract": "^8.0.2",
+		"@types/jest": "^29.4.0",
+		"@typescript-eslint/eslint-plugin": "^5.54.0",
+		"@typescript-eslint/parser": "^5.54.0",
+		"eslint": "^8.35.0",
+		"eslint-config-prettier": "^8.6.0",
+		"eslint-plugin-eslint-comments": "^3.2.0",
+		"eslint-plugin-import": "^2.27.5",
+		"eslint-plugin-jsonc": "^2.6.0",
+		"eslint-plugin-prettier": "^4.2.1",
+		"eslint-plugin-sort-destructure-keys": "^1.5.0",
+		"eslint-plugin-sort-exports": "^0.8.0",
+		"eslint-plugin-typescript-sort-keys": "^2.1.0",
+		"eslint-plugin-unicorn": "^45.0.2",
+		"jest": "^29.4.3",
+		"jest-environment-jsdom": "^29.4.3",
+		"jest-preset-angular": "^13.0.0",
+		"stylelint-config-standard": "^30.0.1",
+		"stylelint-config-standard-scss": "^7.0.1",
+		"stylelint-order": "^6.0.2",
+		"stylelint-prettier": "^3.0.0",
+		"ts-node": "^10.9.1",
+		"type-fest": "^3.6.1",
+		"typescript": "~4.9.4"
+	},
+	"name": "frontend",
+	"private": true,
+	"scripts": {
+		"build": "ng build",
+		"i18n:extract": "ngx-translate-extract --input ./src --output src/locale/en.json --key-as-default-value --clean --format namespaced-json && npx eslint src/locale/*.json --fix",
+		"lint": "npm run lint:code; CODE=$?; npm run lint:style && [ $CODE -eq 0 ]",
+		"lint:code": "eslint . --ignore-pattern '!.*rc.json'",
+		"lint:code:fix": "npm run lint:code -- --fix",
+		"lint:fix": "npm run lint:code:fix; npm run lint:style:fix",
+		"lint:style": "stylelint \"./**/*.{css,scss}\"",
+		"lint:style:fix": "npm run lint:style -- --fix",
+		"ng": "ng",
+		"start": "ng serve",
+		"test": "jest",
+		"test:watch": "jest --watch",
+		"watch": "ng build --watch --configuration development"
+	},
+	"version": "0.0.0"
+}
```

Comment:
Maybe we can add a constraint for node & npm version with the "engines" key.

Reply:
We could, but I do not think it is useful; these constraints can already be determined by its dependencies.
Unless we made a specific node script that needs functions from newer (or older) version, it does not seem necessary to add  this constraint.